images/cluster-setup/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY ./install.sh /usr/local/bin/install.sh
 RUN KUBE_VERSION=v1.24.0 && \
     curl -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$KUBE_VERSION/bin/linux/amd64/kubectl" && \
     chmod 755 /usr/local/bin/kubectl
+RUN microdnf install findutils git
 USER 65532:65532
 VOLUME /workspace
 WORKDIR /workspace
-ENTRYPOINT ["/usr/local/bin/install.sh"]
```

Comment:
Why are you removing the entrypoint? I would understand if it was replaced by a command but it is not the case

Reply:
This was necessary so I could make the image work with GitLab CI.  It might be because I'm unfamiliar with it, so I wouldn't look to hard at that specific change.

Reply:
Fixed